specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerSecurityAdminConfiguration.json

```diff
@@ -806,18 +722,37 @@
           "$ref": "./network.json#/definitions/ProxyResource"
         }
       ],
-      "description": "Network admin rule."
+      "required": [
+        "kind"
+      ],
+      "discriminator": "kind",
+      "description": "Network base rule."
+    },
+    "AdminRule": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AdminPropertiesFormat",
+          "x-ms-client-flatten": true,
+          "description": "Indicates the properties of the security admin rule"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseRule"
+        }
+      ],
+      "description": "Network admin rule.",
+      "x-ms-discriminator-value": "AdminRule"
     },
     "AdminPropertiesFormat": {
-      "type": "object",
       "properties": {
         "displayName": {
           "type": "string",
           "description": "A friendly name for the rule."
         },
         "description": {
           "type": "string",
-          "description": "A description for this rule."
+          "description": "A description for this rule. Restricted to 140 chars."
```

Comment:
add maxLength: 140

Reply:
Sorry, I copied this from other place, and we don't have limite here. Will remove it